docs/cip/README.md

```diff
@@ -1775,57 +1717,82 @@ for further discussion.
 
 <a name="feasible-parameters"></a>**Feasible protocol parameters**
 
-The table below documents a set of Leios protocol parameters that provided high
-throughput and reasonably fast settlement in the prototype Haskell and Rust
-simulations of Leios. The exact choice of parameters that would be adopted on
-the Cardano mainnet must be subject to discussion and consideration of
-tradeoffs.
+**Parameter Relationships and Network Assumptions**
+
+The key innovation in this specification is the relationship between the voting
+threshold ($\tau = 75\%$) and the diffusion period ($L_\text{diff}$). The high
+voting threshold ensures that any certified EB is already known to at least 25%
+of honest nodes by the end of $L_\text{vote}$, even assuming 50% adversarial
+stake. This widespread initial knowledge enables the critical network
+assumption: an EB known to >25% of the network will reach all honest parties
+within the additional $L_\text{diff}$ period.
+
+**Example Parameter Calculation**
+
+To illustrate how these relationships translate into concrete parameters,
+consider the following example based on simulated network measurements:
+
+**Given Network Characteristics:**
+
+- $\Delta_\text{hdr} = 1$ slot, $\Delta_\text{RB} = 5$ slots (current Praos
+  observations)
+- $\Delta_\text{EB}^{\text{opt}} = 7$ slots, $\Delta_\text{EB}^{\text{wc}} = 15$
+  slots (EB diffusion times)
+- $\Delta_\text{cpu} = 3$ slots, $\Delta_\text{reapply} = 1$ slot (validation
+  times)
+
+**Timing Parameter Calibration:**
+
+- $L_\text{vote} \approx \Delta_\text{EB}^{\text{opt}} = 7$ slots (ensures
+  honest EBs diffuse optimally, with margin for validation)
+- $L_\text{diff} \geq \Delta_\text{EB}^{\text{wc}} + \Delta_\text{reapply} - \Delta_\text{RB} - L_\text{vote} = 15 + 1 - 5 - 7 = 4$
+  slots (minimum)
+- Setting $L_\text{diff} = 8$ slots provides additional safety margin
```

Comment:
```suggestion
- Setting $L_\text{diff} = 7$ slots provides additional safety margin
```

@pagio recommended 7, and we're using that in the simulations. However, 8 is fine for this example.